CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.2 (2020-02-07)
+
+- Updated the `list-builder` peer dependency to prevent redundant updates to `ListItemModel` and improve performance for the list view grid component. [#77](https://github.com/blackbaud/skyux-list-builder/pull/77)
```

Comment:
I might start the sentence with, "Added support for `@skyux/list-builder@3.6.3`..." to be more specific about which version.

Reply:
Done.